src/iosrv/init.c

```diff
@@ -41,6 +41,8 @@
 #include <daos/placement.h>
 #include "srv_internal.h"
 #include "drpc_internal.h"
+#include <gurt/telemetry_common.h>
+#include <gurt/telemetry_producer.h>
```

Comment:
Just noticed, telemetry_producer.h and _consumer.h don't include telemetry_common.h. If they did, we could include just telemetry_producer.h here.

Reply:
I'll make that change in a later PR.